portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,90 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <title>Utibeno Ekpoudom</title>
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
+    <link
+      href="https://fonts.googleapis.com/css?family=Merienda|Neucha&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+    <section class="hero">
+      <img src="images/portfolio_pic.jpg" alt="Uty" />
+      <h1>Utibeno Ekpoudom</h1>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      <h3>Welcome to my page</h3>
+    </section>
+    <section class="background">
+      <div class="paragraph">
+        <p>
+          Hey guys, my name is Utibeno Ekpoudom, but Uty is fine as well. I am
+          currently an undergrad studying Computer Science at Georgia Tech.
+          While my concentrations are in intelligence and Infonetworks, I do
+          appreciate amazing design, especially when it comes to web
+          development. Therefore, In addition to my core concentrations I do
+          enjoy designing visually stunning website UIs. Apart from that, I do
+          enjoy watching TV Show, Altered Carbon, GoT, The Expanse etc. As you
+          can tell by my examples I love SciFi, be it shows or movies.
+        </p>
+      </div>
+      <div class="gallery">
+        <img src="images/gatech.jpg" alt="gatech" />
+        <img src="images/nigerianFlag.jpg" alt="" />
+        <img src="images/bugattiDivo.jpg" alt="" />
+        <img src="images/gameOfThrones.jpg" alt="" />
+        <img src="images/tonyStark.jpg" alt="" />
+        <img src="images/alteredCarbon.jpg" alt="" />
```

Comment:
All these images need to be added to the repo. Looks like there are some more below, as well